pkg/controller/elasticsearch/remotecluster/remoteca/controller.go

```diff
@@ -135,6 +123,18 @@ func doReconcile(
 		return reconcile.Result{}, err
 	}
 
+	enabled, err := r.licenseChecker.EnterpriseFeaturesEnabled()
+	if err != nil {
+		return defaultRequeue, err
+	}
+	if !enabled && len(expectedRemoteClusters) > 0 {
+		log.V(1).Info(
```

Comment:
Is it intentional to log this at debug level?

Reply:
Yes, we already have the same log during the API call